countfiles/utils/file_handlers.py

```diff
@@ -53,6 +55,7 @@ def _has_hidden_attribute(filepath):
     Adapted from Jason R. Coombs (https://stackoverflow.com/a/6365265/6167478)
     """
     try:
+        # unresolved reference 'unicode'?
```

Comment:
I believe this is Python 2 remainings. I can't test it in my system, but probably, since file path should be already a string and by default on Python 3 strings are Unicode, we should be able to simply omit the `unicode()` call, like this:

```
         attrs = ctypes.windll.kernel32.GetFileAttributesW(filepath)
```

Can you please check if it is like so?

Reply:
Yes, it's for Python 2.

Reply:
If it works as I suggested above, could you please update your pull request by removing the `unicode()` call?

Reply:
Done